From 1b4c2c9ebc8d098e336f032c81a19d100175902c Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Thu, 29 Jan 2015 15:58:16 -0500
Subject: [PATCH 050/315] display: use bitfield for booleans in priv structure

We're going to be adding more than a couple, so
switch to a bitfield.

https://bugzilla.gnome.org/show_bug.cgi?id=744764
---
 daemon/gdm-display.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/daemon/gdm-display.c b/daemon/gdm-display.c
index cab096bb..b7638512 100644
--- a/daemon/gdm-display.c
+++ b/daemon/gdm-display.c
@@ -64,7 +64,6 @@ struct GdmDisplayPrivate
         gsize                 x11_cookie_size;
         GdmDisplayAccessFile *access_file;
 
-        gboolean              is_local;
         guint                 finish_idle_id;
 
         GdmSlave             *slave;
@@ -74,8 +73,9 @@ struct GdmDisplayPrivate
         GdmDBusDisplay       *display_skeleton;
         GDBusObjectSkeleton  *object_skeleton;
 
-        gboolean              is_initial;
-        gboolean              allow_timed_login;
+        guint                 is_local : 1;
+        guint                 is_initial : 1;
+        guint                 allow_timed_login : 1;
 };
 
 enum {
-- 
2.15.1

